nix/sources.json

```diff
@@ -53,10 +65,10 @@
         "homepage": null,
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "6c2318e451fc0eaf338fb461d9bfcc99869de758",
-        "sha256": "1djwkvjnn26v0xw1swwh7dgmi0yl0b1kb8kansrdwk0jhhqbl7zq",
+        "rev": "89abc99d5b6351b2984c6126ef3bda7929905cb9",
```

Comment:
I've bumped `nixpkgs`, since the older version doesn't contain `rustPlatform.bindgenHook` and I wanted to have a test as close to the original report as possible.